docs/src/content/docs/ja/reference/overrides.md

```diff
@@ -0,0 +1,356 @@
+---
+title: オーバーライド
+description: Starlightのオーバーライドでサポートされているコンポーネントとコンポーネントプロパティの概要。
+tableOfContents:
+  maxHeadingLevel: 4
+---
+
+Starlightの[`components`](/ja/reference/configuration#components)設定オプションに置き換え対象のコンポーネントへのパスを指定することで、Starlightの組み込みコンポーネントをオーバーライドできます。このページでは、オーバーライド可能なすべてのコンポーネントと、GitHub上にあるコンポーネントのデフォルト実装へのリンクの一覧を記載しています。
+
+[コンポーネントのオーバーライドガイド](/ja/guides/overriding-components/)も参照してください。
+
+## コンポーネントprops
+
+すべてのコンポーネントは、現在のページに関する情報を含んでいる、標準の`Astro.props`オブジェクトにアクセスできます。
+
+カスタムコンポーネントに型を付けるには、Starlightから`Props`型をインポートします。
+
+```astro
+---
+// src/components/Custom.astro
+import type { Props } from '@astrojs/starlight/props';
+
+const { hasSidebar } = Astro.props;
+//      ^ type: boolean
+---
+```
+
+これにより、`Astro.props`にアクセスする際、オートコンプリートと型が有効になります。
+
+### Props
+
+Starlightは、以下のpropsをカスタムコンポーネントに渡します。
+
+#### `dir`
+
+**Type:** `'ltr' | 'rtl'`
+
+ページの書字方向。
+
+#### `lang`
+
+**Type:** `string`
+
+このページのロケールのBCP-47言語タグ。たとえば`en`、`zh-CN`、`pt-BR`など。
+
+#### `locale`
+
+**Type:** `string | undefined`
+
+言語が配信されるベースパス。ルートロケールスラグの場合は`undefined`となります。
+
+#### `slug`
+
+**Type:** `string`
+
+コンテンツファイル名から生成されたページのスラグ。
+
+#### `id`
+
+**Type:** `string`
+
+コンテンツファイル名に基づくページの一意のID。
+
+#### `isFallback`
+
+**Type:** `true | undefined`
+
+このページが現在の言語で未翻訳であり、デフォルトロケールのフォールバックコンテンツを使用している場合は`true`となります。多言語サイトでのみ使用されます。
+
+#### `entryMeta`
+
+**Type:** `{ dir: 'ltr' | 'rtl'; lang: string }`
+
+ページコンテンツのロケールメタデータ。ページがフォールバックコンテンツを使用している場合、トップレベルのロケール値とは異なる場合があります。
+
+#### `entry`
+
+現在のページのAstroコンテンツコレクションのエントリー。`entry.data`には、現在のページのフロントマターの値が含まれます。
+
+```ts
+entry: {
+  data: {
+    title: string;
+    description: string | undefined;
+    // etc.
```

Comment:
Just to double check, is this fine untranslated or something missed?

Reply:
@HiDeoo I intentionally left the expression 'etc.' as it is, thinking that most Japanese people would understand it. However, I now believe it might be more natural to translate it, so I will update it!